src/main/java/org/elasticsearch/action/update/UpdateHelper.java

```diff
@@ -240,7 +240,7 @@ public Result prepare(UpdateRequest request, IndexShard indexShard) {
             deleteRequest.operationThreaded(false);
             return new Result(deleteRequest, Operation.DELETE, updatedSourceAsMap, updateSourceContentType);
         } else if ("none".equals(operation)) {
-            UpdateResponse update = new UpdateResponse(getResult.getIndex(), getResult.getType(), getResult.getId(), getResult.getVersion(), false);
+            UpdateResponse update = new UpdateResponse(getResult.getIndex(), getResult.getType(), getResult.getId(), getResult.getVersion(), false, true);
```

Comment:
instead of boolean if we can send the opcode in the response, it would be more useful